src/store/errors/getters.ts

```diff
@@ -0,0 +1,12 @@
+import { GetterTree } from 'vuex';
+
+import { RootStateInterface } from '@/store/types';
+import { ErrorsStateInterface } from '@/store/errors/types';
+
+export const getters: GetterTree<ErrorsStateInterface, RootStateInterface> = {
+  getErrorMessage: (state) => {
+    const errorMessage = state.error?.errorMessage;
+    const errorStatus = state.error?.errorStatus;
+    return { errorMessage, errorStatus };
```

Comment:
If there (line 8) is no error in errorMessage for example, what will be returned in line 10 instead of errorMessage? Is it okay / do no extra errors appear?